src/components/Restoration/ExtendTimeLimit.vue

```diff
@@ -30,16 +29,14 @@
       :class="{ 'invalid-section': !getApprovalTypeValid }"
       v-if="approvalType === ApprovalTypes.VIA_COURT_ORDER"
     >
-      <template>
```

Comment:
Fixed "no empty template".

Reply:
What does this mean exactly Sev? Why was lint complaining about this?

Reply:
Until now, I like the use of an empty template tag so the code within could be code-folded -- a nice way to collapse things you don't care about when looking through a file.

However, a new lint rule says we shouldn't have empty template blocks. I could have disabled that rule but instead I chose to fix it. Here, there was only 1 component in the block, so I removed the block. In other places, I changed the block to a div instead (this could be risky as divs are rendered and CSS could select them, but we usually select by id or classname so I think we're OK).

Reply:
Oh okay. Noted.

Question, if we had something like `<template v-if="hasBusinessNameChanged">` and inside this template, we also have only 1 component, Will this new lint rule also complain about this?

Reply:
No, because then the template wouldn't be empty 😉

Reply:
Ooo okay! Makes sense. Thanks Sev!